onchain/src/tests/art_peace.cairo

```diff
@@ -425,3 +425,209 @@ fn deposit_reward_test() {
         art_peace_token_balance == reward_amount, "reward wrongly distributed when adding template"
     );
 }
+
+#[test]
```

Comment:
Could you move these to a separate `color_voting.cairo` file in the tests dir?

Reply:
Done